smoke-test/smoke.sh

```diff
@@ -21,6 +21,7 @@ pip install -r requirements.txt
 
 echo "DATAHUB_VERSION = $DATAHUB_VERSION"
 DATAHUB_TELEMETRY_ENABLED=false datahub docker quickstart --quickstart-compose-file ../docker/quickstart/docker-compose-without-neo4j.quickstart.yml --dump-logs-on-failure
+#DATAHUB_TELEMETRY_ENABLED=false datahub docker quickstart --standalone_consumers --build-locally --dump-logs-on-failure
```

Comment:
I'm personally okay with leaving this in as a way to easily run it locally. Might even want to create a separate file for this and have it run as a part of releases at some point or at least run some of the short tests against an instance with standalone consumers.

Reply:
I kept this to change after this PR is merged so that we start using standalone consumers during our smoke tests. I think we should have that as part of our CI.